src/players/gateways/players.gateway.ts

```diff
@@ -34,6 +38,10 @@ export class PlayersGateway
 
   constructor(private events: Events, private playersService: PlayersService) {}
 
+  async emit(event: WebsocketEvent, payload: Serializable<PlayerDto>) {
```

Comment:
we should make this a superclass and inherit this method from that.

should be done in a follow up